src/runtime-rs/config/configuration-remote.toml.in

```diff
@@ -0,0 +1,302 @@
+# Copyright (c) 2019-2022 Alibaba Cloud
+# Copyright (c) 2019-2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_REMOTE_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+
+[hypervisor.remote]
+remote_hypervisor_socket = "/run/peerpod/hypervisor.sock"
+remote_hypervisor_timeout = 600
```

Comment:
Please document these settings similar to how existing setting are documented.  What is the remote hypervisor socket, who is the expected remote peer?  When is the time-out applied (connection setup? keep-alive?), which units is it given in?

Reply:
These are from the runtime of Go, and I want this configuration file to be compatible with the Go's.
And I have filled in the documents in my latest submission.